app/controllers/jobs_controller.rb

```diff
@@ -78,6 +78,6 @@ def set_group
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def job_params
-      params.require(:job).permit(:name, :status, :duration, :message)
+      params.require(:job).permit(:name, :status, :duration, :message, :frequency)
```

Comment:
In testing, "frequency" doesn't appear to be required, and if omitted, bad things happen.